algorithms/kernel/adaboost/adaboost_model_fpt.cpp

```diff
@@ -38,17 +38,14 @@ namespace interface2
  * \DAAL_DEPRECATED_USE{ Model::create }
  */
 template <typename modelFPType>
-DAAL_EXPORT Model::Model(size_t nFeatures, modelFPType dummy) :
-    _nFeatures(nFeatures),
-    _models(new data_management::DataCollection())
+DAAL_EXPORT Model::Model(size_t nFeatures, modelFPType dummy) : _nFeatures(nFeatures), _models(new data_management::DataCollection())
```

Comment:
why here we have initialization in one row, but in algorithms/kernel/adaboost/adaboost_model.cpp on separate rows?